Remove sleeps.

HeMan opened the pull request:
- the sleeps is not needed.

HeMan committed:
Remove sleeps.

- the sleeps is not needed.

OttoWinter commented:
I mean the sleeps have no use code-wise, they were solely intended to make the wizard feel a bit more "interactive" and a bit more like an actual wizard or actual person. The aim was to make the onboarding experience as friendly as possible. Of course the drawback of these sleeps is that going through the wizard is a bit slower. However, I think that nobody's going to go through the wizard more than say 5 times anyway. 

Maybe the sleep periods are a bit too long, but in general it's nicer to have the text appear in succession and not have a wizard that just starts with a huge text blob that you have to parse through first. Don't get me wrong, I'm also kind of on the edge of having these sleeps in the wizard or not, but ATM I think they're actually good. I'm open to suggestions/reasons why they should not be there though.

HeMan committed:
Make sleep depending on environment variable.

- set QUICKWIZARD to true to disable sleeps.

HeMan commented:
I added a environment variable called QUICKWIZARD that disables the sleeps.

OttoWinter commented on esphomeyaml/wizard.py:
```diff
@@ -71,6 +70,13 @@
 """
 
 
+if os.environ['QUICKWIZARD']:
```
This results in a `KeyError` if `QUICKWIZARD` is not specified in the environment:

<img width="784" alt="screen shot 2018-04-11 at 21 40 49" src="https://user-images.githubusercontent.com/6833237/38639303-099d6492-3dd1-11e8-8121-9b1bbd94620f.png">

This should work:

```yaml
if os.getenv('QUICKWIZARD', False):
    def sleep(time):
        pass
else:
    from time import sleep
```

Also, I mean I don't really care, but the other environment variable I've defined for OTAs, `ESPHOMEYAML_OTA_HOST_PORT`, has the prefix `ESPHOMEYAML_` so I'd prefer if this environment variable would also start with that prefix for consistency.

HeMan replied:
Lol! It shows how I tested it!

OttoWinter reviewed (commented):
Using environment variables is an excellent solution 👍

HeMan committed:
Changed env-name and made it work without env.

- It only worked when environment variable was defined. Now it works
  with variable unset, which should be the normal case.
- Added ESPHOMEYAML_ as prefix so it's ESPHOMEYAML_QUICKWIZARD.

OttoWinter reviewed (approved):
Looks good! Thanks 🐳👍
